docs/source/tutorials_source/package/tutorial_checkpoint_finetuning.py

```diff
@@ -0,0 +1,251 @@
+import torch
+import torch.nn as nn
+from lightning.pytorch.loggers import TensorBoardLogger
+from torchvision import transforms
+
+from lightly.transforms.utils import IMAGENET_NORMALIZE
+
+# %%
+# Downloading Model Checkpoint
+# -----------------------------
+#
+# Let's use the resnet50 model pre-trained on ImageNet1k using
+# the `SimCLR <https://arxiv.org/abs/2002.05709>`_ method. We will
+# download the model weights from the S3 bucket and store them in memory
+#
+# .. code-block:: console
+#
+#   wget -O weights.ckpt https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_simclr_2023-06-22_09-11-13/pretrain/version_0/checkpoints/epoch%3D99-step%3D500400.ckpt
+#
+# You can browse other model checkpoints at :ref:`lightly-benchmarks`.
+
+# %%
+# Tensorboard Logger
+# ---------------------------
+#
+
+tb_logger = TensorBoardLogger("tb_logs", name="lightly_finetuning")
+
+
+# %%
+# Configuration
+# -------------
+# Let's set the configuration parameters for our experiments.
+#
+# We use a batch size of 32 and an input size of 128.
+#
+# We only train for 5 epochs because the focus of this tutorial is on
+# finetuning lightly checkpoints.
+
+learning_rate = 0.001
+num_workers = 8
+batch_size = 3
+input_size = 128
+
+seed = 42
+num_train_epochs = 5
+
+# use cuda if possible
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+# %%
+# Setup data augmentations and loaders
+# -------------------------------------
+# For this tutorial, we'll use the Food-101 dataset of 101 food categories
+# with 101,000 images. For each class, 250 manually reviewed test images are
+# provided as well as 750 training images. On purpose, the training images
+# were not cleaned, and thus still contain some amount of noise. This comes
+# mostly in the form of intense colors and sometimes wrong labels.
+# All images were rescaled to have a maximum side length of 512 pixels.
+#
+# We will also use some minimal augmentations for the train and test subsets.
+# To learn more about data pipelines in LightlySSL you can refer to :ref:`input-structure-label`
+# and to learn more about the different augmentations and learned invariances please refer to
+# :ref:`lightly-advanced`.
+
+from torch.utils.data import DataLoader
+from torchvision.datasets import Food101
+
+num_classes = 101
+
+
+# Training Transformations
+train_transform = transforms.Compose(
+    [
+        transforms.RandomResizedCrop(input_size),
+        transforms.RandomHorizontalFlip(),
+        transforms.ToTensor(),
+        transforms.Normalize(
+            mean=IMAGENET_NORMALIZE["mean"],
+            std=IMAGENET_NORMALIZE["std"],
+        ),
+    ]
+)
+train_dataset = Food101(
+    "datasets/food101", split="train", download=True, transform=train_transform
+)
+
+train_dataloader = DataLoader(
+    train_dataset,
+    batch_size=batch_size,
+    shuffle=True,
+    drop_last=True,
+    num_workers=num_workers,
+)
+
+# Test Transformations
+test_transform = transforms.Compose(
+    [
```

Comment:
Optional: I think it would make sense to call everything `val` instead of `test` to also be consistent with pytorch lightning.